### DISCUSSION TITLE
How to uninstall git?

### DISCUSSION THREAD

**Discussion by bbannon** [git-for-windows/git#3129] [Q&A] [answered]
I no longer need git. How do I uninstall it?

**Comment by PhilipOakley** [comment, marked as answer]
On Win10, you can use the start button and scroll down the app list to git, expand it and right click any of the apps, such as `Git Bash` to see the uninstall option (I'm not sure if this Windows option doe the whole Git for Windows system or just the one particular part...).

Or, more directly, go to the `C:\Program Files\Git\unins001.exe` which will uninstall the complete Git for Windows. It's unfortunate that the 8.3 format means it can't fully spell out `unistall001.exe`

**Reply by dscho** [reply]
There should also be an "Add or Remove Program" entry somewhere in the Start menu.

**Reply by PhilipOakley** [reply]
> There should also be an "Add or Remove Program" entry somewhere in the Start menu.

Hi @dscho (for completeness) 
I'm not seeing an "Add or Remove Program" entry on my Win10(Home). When I click the Start button, it pops-up the scrollable list of installed programmes, with apps tiles slightly to the right of the scroll list. 

There is also the 'Settings' gear icon just above the start button (after clicking) which then gives a settings dialog box which includes the "Apps (uninstall, defaults, optional features)" clickable selection (and more clicking)...

Summary: I miss the good old "Add or Remove Program" entry too.  😃

**Reply by dscho** [reply]
TBH I usually hit the Windows key, then type "remove" and _then_ I see the entry.
